service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -45,17 +46,17 @@
 
     @BeforeEach
     void setup() {
-        given(organisationService.findOrganisation()).willReturn(POPULATED_ORGANISATION);
+        given(organisationService.findOrganisation()).willReturn(Optional.of(POPULATED_ORGANISATION));
         given(authTokenGenerator.generate()).willReturn(SERVICE_AUTH_TOKEN);
-        given(organisationApi.findOrganisationById(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN))
+        given(organisationApi.findUserOrganisation(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN))
```

Comment:
If you think that the service mock is redundant rather than the api mock I think that this stubbing should be moved to the relevant test. There are only two tests and it is being overridden in one of them.